front/src/Api/IframeListener.ts

```diff
@@ -665,6 +677,35 @@ class IframeListener {
         });
     }
 
+    sendChatVisibilityToChatIframe(visibility: boolean) {
+        this.postMessageToChat({
+            type: "chatVisibility",
+            data: {
+                visibility,
+            },
+        } as IframeResponseEvent);
+    }
+    sendSettingsToChatIframe() {
```

Comment:
The notification are sent only when the user change it. But I think that we need to sen them with user data?
Because if the user don't change settings and it's rather unsual, the notification settings will never defined in chat.

Reply:
You're right, I send this settings when the user data are sent now.